string src_robot
int32 src_robot_id
float64 x
float64 y
